content/en/docs/gsoc/_index.md

```diff
@@ -132,12 +132,12 @@ This task can be nicely paired with the jukebox one.
 #### Steps
 
 - Implementing the glue between Alexa/Google home and the Subsonic API
-- Implementing the frontend part to configure this machinerie
+- Implementing the frontend part to configure this machinery
 
 #### Details
 
 * Skill level: Hard
-* Required abilities: Go, having an Alexa/Google Home is a plus, but worse case we can send the student a device.
+* Required abilities: Go, having an Alexa/Google Home is a plus, but in the worse case we can send the student a device.
```

Comment:
No need to add `in the` here.

Reply:
@Himanshunitrr Is this resolved?

Reply:
According to me ```in the``` made more sense but I am not expert in english so @jvoisin must be right. So it's resolved.

Reply:
Ok, but you need to remove it from your PR. There's also the other change requested. Please change these two things so I can merge it. Thanks!

(By the way, nobody here is expert in English :P)